mail_cleanup/models/mail_cleanup.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# © 2015-2016 Matthieu Dietrich (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from openerp import api, fields, models
+from dateutil.relativedelta import relativedelta
+
+from openerp.addons.server_environment import serv_config
+
+_logger = logging.getLogger(__name__)
+
+
+class FetchmailServer(models.Model):
+    """Incoming POP/IMAP mail server account"""
+    _inherit = 'fetchmail.server'
+
+    @api.multi
+    def _get_cleanup_conf(self):
+        """
+        Return configuration
+        """
+        for fetchmail in self:
+            global_section_name = 'incoming_mail'
+
+            # default vals
+            config_vals = {'cleanup_days': False,
+                           'purge_days': False,
+                           'cleanup_folder': False}
+            if serv_config.has_section(global_section_name):
+                config_vals.update(serv_config.items(global_section_name))
+
+            custom_section_name = '.'.join((global_section_name,
+                                            fetchmail.name))
+            if serv_config.has_section(custom_section_name):
+                config_vals.update(serv_config.items(custom_section_name))
+
+            # convert string values to integer
+            if config_vals['cleanup_days']:
+                config_vals['cleanup_days'] = int(config_vals['cleanup_days'])
+            if config_vals['purge_days']:
+                config_vals['purge_days'] = int(config_vals['purge_days'])
+
+            for field in ['cleanup_days', 'purge_days', 'cleanup_folder']:
+                fetchmail[field] = config_vals[field]
+
+    cleanup_days = fields.Integer(
+        compute=_get_cleanup_conf,
+        string='Expiration days',
+        help="Number of days before marking an e-mail as read")
+
+    cleanup_folder = fields.Char(
+        compute=_get_cleanup_conf,
+        string='Expiration folder',
+        help="Folder where an e-mail marked as read will be moved.")
+
+    purge_days = fields.Integer(
+        compute=_get_cleanup_conf,
```

Comment:
Please use string for computation declarations in order to allow inherited changes: `compute='_get_cleanup_conf',`